@kiva/kv-components/src/vue/KvLendCta.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div class="tw-whitespace-nowrap">
 		<kv-cart-pill
```

Comment:
We actually want to remove `KvCartPill` from the CTA component. The pill component is used in the checkout page in `ui`, but it's currently used for the CTA.

Reply:
We can also remove the `showPill` usage in `cms-page-server` where we pass in `false` on the impact dashboard.

Reply:
Will remove in another brief simple ticket!